app/(dashboard)/mailboxes/[mailbox_slug]/[category]/conversation/issueAssignButton.tsx

```
@@ -0,0 +1,103 @@
+"use client";
+
+import { GitBranch, X } from "lucide-react";
+import { useState } from "react";
+import { toast } from "sonner";
+import { Button } from "@/components/ui/button";
+import { Select, SelectContent, SelectItem, SelectTrigger, SelectValue } from "@/components/ui/select";
+import { cn } from "@/lib/utils";
```

ESLint

[error] 'cn' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)